crates/unftp-sbe-gcs/src/response_body.rs

```diff
@@ -71,6 +73,11 @@ impl Item {
 
         Ok(Fileinfo { path, metadata })
     }
+
+    pub(crate) fn to_md5(&self) -> Result<String, Error> {
+        let md5 = base64::decode(&self.md5_hash).map_err(|e| Error::new(ErrorKind::LocalError, e))?;
+        Ok(md5.iter().map(|b| format!("{:02x}", b)).collect())
```

Comment:
I guess we should document whether the MD5 should be upper or lower case.

Reply:
Not sure what is the standard tbh. I haven't found any RFC regarding this. It is truly a site specific extension. However it should most likely be lowercase, as the md5sum command in Unix formats it. I will document this.

Reply:
I think if we just choose one and stick to it.

Reply:
added a little documentation